_includes/tutorials/kafka-consumer-application/confluent/markup/dev/make-config-file.adoc

```diff
@@ -0,0 +1,38 @@
+Then create a development configuration file at `configuration/dev.properties` with your Confluent Cloud information (specifically fill in the `bootstrap.servers`, `sasl.jaas.config`, `basic.auth.user.info`, and `schema.registry.url` configs):
```

Comment:
1. Should we direct users to the `Clients` page to auto-generate this config file?
2. Should this be part of `shared` step (assuming other tutorials will use the same thing)?  Note, it would require:

a) modifying the original KT to split this out
b) doing the concat on different properties files.

Reply:
hm I took a crack at splitting this up into a shared section. Then ask the user to append the KT specific configs to the properties file created. Open to other ideas here. Commit https://github.com/confluentinc/kafka-tutorials/pull/786/commits/c21a720da2a3d4f70094211cd2a87d5e0cd163e3